ipwb/util.py

```diff
@@ -144,6 +150,10 @@ def setIPWBReplayIndexPath(cdxj):
 
 def getIPWBReplayIndexPath():
     ipfsJSON = readIPFSConfig()
```

Comment:
`ipfsJSON` is not defined prior to that conditional. Removing the first line of that function will produce a NameError.